third_party/2and3/werkzeug/utils.pyi

```diff
@@ -31,7 +31,15 @@ def format_string(string, context): ...
 def secure_filename(filename): ...
 def escape(s, quote: Optional[Any] = ...): ...
 def unescape(s): ...
-def redirect(location, code: int = ..., Response: Optional[Any] = ...) -> Response: ...
+
+# 'redirect' returns a werkzeug Response, unless you give it
+# another Response type to use instead.
+ResponseClass = TypeVar("ResponseClass", bound=Response)
```

Comment:
Type var names should be very short and marked as private. So just `_R` or `_RC` or something similar would be better name. This is also why CI fails.